src/reducer.js

```diff
@@ -0,0 +1,40 @@
+const initialState = {
+  id: 100,
+  restaurant: {},
```

Comment:
```suggestion
  restaurant: initialRestaurant,
```
initialRestaurant 객체를 하나 추가해서 name, category, address를 관리해주면 좋을 것 같아요~